lib/start.js

```diff
@@ -97,7 +97,13 @@ const start = (passthroughArgs, buildConfig = config.defaultBuildConfig, options
   let outputPath = options.output_path
   if (!outputPath) {
     if (process.platform === 'darwin') {
-      outputPath = path.join(config.outputDir, config.macAppName() + '.app', 'Contents', 'MacOS', config.macAppName())
+      let outputDir = config.outputDir
+      if (config.shouldSign()) {
+        outputDir = path.join(outputDir, config.mac_signing_output_prefix)
+      }
+      outputPath = path.join(outputDir,
+                             'Brave\\ Browser\\ Development.app', 'Contents', 'MacOS',
```

Comment:
macAppName has been replaced with "Brave Browser Development", what about Beta etc?

Reply:
this script isn't used by release builds and it was simpler to just always make it Development